### PR TITLE
v93.4.0

### PR DESCRIPTION
**Pull request by galiak11**

### CODE REVIEW

**Comment by galiak11** [comment]
@googlebot I fixed it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+This minor release adds a new Filled and Outlined TextFields that will eventually replace MDCTextField's filled and outlined controllers. Additionally, it fixes dynamic type issues for multiple components, and removes deprecated themers.
+
+## New features
+
+### TextFields
+
+* [Add filled textfields (#8674)](https://github.com/material-components/material-components-ios/commit/f2e5fe4fbd17735858b1897765ce6c4f55147cb9) (Andrew Overton)
+* [Add outlined textfield (#8705)](https://github.com/material-components/material-components-ios/commit/cfc699b1992c56e2e67cfc082e568863ac80da18) (Andrew Overton)
+
+## API changes
```
Comment (romoore) [inline CHANGELOG.md:12]: "API Changes" is usually that autogenerated bit, but it seems none was included. I think you can just include these sections under "Component Changes" below.

Comment (fluxminds) [reply CHANGELOG.md:12]: Agreed

**Review by fluxminds — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (galiak11) [reply CHANGELOG.md:12]: Done

**Pull request merged**